modules/metastorage/src/main/java/org/apache/ignite/internal/metastorage/server/persistence/RocksDbKeyValueStorage.java

```diff
@@ -311,6 +318,12 @@ protected void createDb() throws RocksDBException {
         }
     }
 
+    private void notifyRevisionUpdate() {
```

Comment:
We should specify that this method must be called under the write lock

Reply:
You need to specify that the notification will be at the end of the recovery.

Reply:
It is called every time, not only on recovery. But listener will be invoked only on recovery.